### DISCUSSION TITLE
Ability to group/ungroup containers into another parent container

### DISCUSSION THREAD

**Discussion by enricofrascati** [elementor/elementor#25241] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

This is an operation that I do often, moving containers or widget inside and outside other containers. And since the structure panel UI is slow and sometimes become unreliable, it becomes really frustrating.  I feel like, as in figma, it would be much faster to have the option to group multiple containers/widget in just one click, and maybe ungroup them if needed.

### Describe the solution you'd like.

I' d like to select multiple elements in the layers panel and group them under a new container in just one click, with a group/ungroup button in the structure panel.

### Describe alternatives you've considered.

Probably an alternative could be the option not only to copy/paste a widget or a container, like now, but also to cut/paste elements around. And so moving elements around the structure  using just the keyboard. I don't want to have to rely only on the structure panel or on the drag and drop interface of the editor, because with complex page structure and nested containers become unreliable and slow. And probably, from my point of view, it's one of the worst parts of the Elementor experience. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by dbevec** [comment]
God, yes please
